apps/fipsinstall.c

```diff
@@ -372,17 +395,25 @@ int fipsinstall_main(int argc, char **argv)
         case OPT_OUT:
             out_fname = opt_arg();
             break;
+        case OPT_PEDANTIC:
+            fips_opts = pedantic_opts;
```

Comment:
Dont like this copy here much.. Why not have a variable that points to constant tables?

Reply:
Because the table needs to be non-const unless this option is set.  Other options alter it's contents.

The structure occupies one byte currently (probably rounds up to four), the copy isn't an overhead.

Reply:
I could `memset(fips_opts, 255, sizeof(fips_opts))` which avoids the copy but relies on all FIPS options being set bits/flags.

Reply:
Non leave as it is - I missed the const on the first table but not the second.